CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 	- Removed "last user" cache to avoid overwriting anonymous user cache during init [SDKCF-6409]
 - Improvements:
 	- Created new test scheme to combine unit test and UI test results [SDKCF-6356]
-    - Added more unit test to increase code coverage [SDKCF-6379]   
+    - Added more unit test to increase code coverage [SDKCF-6379]
```

Comment:
nit: this line still starts with 4 spaces instead of a tab character like in the lines above